integration_test/third_party_apps_data/applications/mongodb/debian_ubuntu/install

```diff
@@ -0,0 +1,42 @@
+set -e
+
+sudo apt-get install -y gnupg wget
+wget -qO - https://www.mongodb.org/static/pgp/server-5.0.asc | sudo apt-key add -
+
+source /etc/os-release
+case $VERSION_ID in
+  20.04)
+    echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu focal/mongodb-org/5.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-5.0.list
```

Comment:
You can use `$(lsb_release -sc)` to get the codename and switch on `$(lsb_release -si)` which is `Ubuntu` or `Debian` to simplify the case and futureproof, though I guess you still want to hardcode Debian 11 for now since it needs buster packages.